sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/OoyalaSkinPlayerObserver.java

```diff
@@ -71,7 +71,7 @@ private void bridgeStateEmbedCodeNotification() {
   }
 
   private void bridgeCurrentItemChangedNotification() {
-    WritableMap params = BridgeMessageBuilder.buildCurrentItemChangedParams(_player, _layoutController.width, _layoutController.height);
+    WritableMap params = BridgeMessageBuilder.buildCurrentItemChangedParams(_player, _layoutController.width, _layoutController.height, _layoutController.currentVolume);
```

Comment:
I'm fine with you modifying the signature, but instead of calling on the global variable, use the static getCurrentVolume